client/src/components/create-assignment/ParentAssignment.js

```diff
@@ -18,12 +18,12 @@ import Assignment from './Assignment';
 import assignmentServices from '../../services/assignments';
 import formulasService from '../../services/formulas';
 
-// An Assignmnet component with subassignments and a formula
+// An Assignmnet component with subAttainments and a formula
```

Comment:
Same here, one assignment left to change

Reply:
Same here.